crates/gitbutler-core/src/ops/oplog.rs

```diff
@@ -73,16 +73,21 @@ impl Oplog for Project {
         let repo = git2::Repository::init(repo_path)?;
 
         let vb_state = self.virtual_branches();
+
+        // grab the target tree sha
         let default_target_sha = vb_state.get_default_target()?.sha;
+        let default_target = repo.find_commit(default_target_sha.into())?;
+        let target_tree_oid = default_target.tree_id();
 
+        // figure out the oplog parent
         let oplog_state = OplogHandle::new(&self.gb_dir());
         let oplog_head_commit = match oplog_state.get_oplog_head()? {
             Some(head_sha) => match repo.find_commit(git2::Oid::from_str(&head_sha)?) {
-                Ok(commit) => commit,
-                Err(_) => repo.find_commit(default_target_sha.into())?,
+                Ok(commit) => Some(commit),
+                Err(_) => None, // cant find the old one, start over
             },
-            // This is the first snapshot - use the default target as starting point
-            None => repo.find_commit(default_target_sha.into())?,
```

Comment:
We don't want to do this, the way this was implemented, we would use the target sha as the first parent we build the oplog on top of. We would rather have the first oplog entry be parentless so we know when the history started.